app/javascript/app/components/cookie-consent/cookie-consent-styles.scss

```diff
@@ -3,13 +3,13 @@
 .container {
   position: fixed;
   bottom: 0%;
-  z-index: 15;
+  z-index: $z-index-cookie-consent;
   width: 100%;
   padding: 10px 15px;
-  background-color: #0A1941;
+  background-color: #0a1941;
   font-family: $font-family-1;
   font-size: 14px;
-  color: #F4F5F6;
+  color: #f4f5f6;
```

Comment:
Please add this color to settings.scss or maybe replace it with a similar one that we already have, maybe `$light-gray`?